src/filesystem/impls/appshell/node/FileWatcherDomain.js

```diff
@@ -99,7 +99,8 @@ function watchPath(path, ignored) {
             ignoreInitial: true,
             ignorePermissionErrors: true,
             followSymlinks: true,
-            ignored: ignored
+            ignored: ignored,
+            usePolling: process.platform === "win32"
```

Comment:
I just saw there were already some issues filed against chokidar and the solution seems to use the `usepolling` option 😞 
It seems to me that even VSCode uses its own watcher system on Windows: https://github.com/Microsoft/vscode-filewatcher-windows

By the way on unix they seems to use these options: https://github.com/Microsoft/vscode/blob/0d658f2c0308c770b90df5a73816f0c045817689/src/vs/workbench/services/files/node/watcher/unix/chokidarWatcherService.ts#L30